src/components/SignIn/index.jsx

```diff
@@ -85,7 +85,7 @@ function SignIn() {
 
       function handleError(error) {
         confirmAlert({
-          message: `${error.response.data.message ?? 'Something went wrong'}. Please try again.`,
+          message: `${error.response.data?.message ?? 'Something went wrong'}. Please try again.`,
```

Comment:
O erro "cannot read properties of 'undefined' (reading 'data')" vinha daqui, só precisei adicionar um optional chaining operator mesmo pra solucionar